app.ts

```diff
@@ -35,18 +31,13 @@ import { testWebhookReceiver } from "./testWebhookReceiver.ts";
 
 async function prepareAndMain() {
```

Comment:
While prisma generate is already required to execute app.ts, I'm not sure prepareAndMain really required to exist just to execute prisma generate once more. If you still think it's better to exist, I think now it needs to execute prisma migrate, rather than prisma generate. Or it could be helpful if emitter and observer(or any module imports prisma-shim.ts) is dynamically imported in main().

Reply:
The intent was to have stuff ready before the components were run (as app.ts is intended for development, and I'm thinking of changing the name to dev.ts.) Also, prisma migrate doesn't work at the moment, so I'll change it to `db push`. I did try dynamic import, but it seems like even if the modules are dynamically imported, deno will still not be able to find generated code, maybe because it's not compiled from the beginning. 🤔

Reply:
Amended!